Add case study 2

WenheLI opened the pull request.

WenheLI committed:
Add case study 2

shiffman commented on CS02-React&Redux.md:
Could you include some information about the licensing controversy here?

https://en.wikipedia.org/wiki/React_(JavaScript_library)#Licensing_controversy

shiffman replied:
I think it's also worth noting that contributors have to sign a "CLA":

https://code.facebook.com/cla

WenheLI replied:
Oh yes, I should mention these on the document!

shiffman commented on CS02-React&Redux.md:
Would you characterize the developer or user community of React as diverse? Are there any efforts or initiatives to welcome / include people from underrepresented groups in tech?

D31F43 replied:
Yes. There is one thing should be mentioned in the world **diversity** as we forgot to tell in class. The **diversity** I understand for *React* is that *React* actually brings web development to the whole stack of hybrid development. It highlights the concept of *ES6 Class* to embrace the combination of OOP programming, which is a breakthrough for it connects **Native App** concept with **Web**. This promotion is largely welcomed by both **Web Community** and **Native App Community** as it simplifies (or more accurately standardizes) the program standard in **Web**. And this is also a very important reason why it has larger community than most of other JS Library communities.

shiffman commented on CS02-React&Redux.md:
Do you think there is any way to estimate the ratio of facebook employees to outside contributors in the contributor community?

WenheLI replied:
Yes for this thing, I think I can write a crawler to calculate the ratio.

shiffman reviewed (commented):
I'd love to hear your thoughts on these comments / questions!

WenheLI committed:
Update CS02-React&Redux.md

Add what story about licensing controversy, diversity and ratio of Facebook employees through GitHub API and Python

shiffman reviewed (approved):
Thanks for the updates!

shiffman commented on CS02-React&Redux.md:
```diff
@@ -26,9 +26,13 @@
 - Facebook
 - Community
     - React: 1240 Contributors
+    - Around 30% of the contributors are from Facebook
```
Interesting! I wonder what it would be by volume. For example are the 70% from outside Facebook making little small changes but the bulk of the work is from facebook. How many commits are from Facebook employees? You don't have to research this, I'm just posing the questions as food for thought.
